core/domain/feedback_services.py

```diff
@@ -37,9 +37,21 @@
 def _create_models_for_thread_and_first_message(
         exploration_id, state_name, original_author_id, subject, text,
         has_suggestion):
-    """Creates a thread and the first message in it.
+    """Create a feedback thread and its first message
 
-    Note that `state_name` may be None.
+    Args:
+        exploration_id: str. The exploration id the thread belongs.
+        state_name: str or None. The state name of the thread
+            which could be None.
+        original_author_id: str. The author id who starts this thread.
+        subject: str. The subject of this thread.
+        text: str. The text of the message being created which is allowed
```

Comment:
That seems odd. Are there any callers that use None? If so, can you point me to them? I feel like we should force this to be a string, but let's see what the usage patterns are like. 

On the other hand, if None ought to be allowed, then we need to change the docstring to reflect that, similar to what is done for the state_name. I don't see why '' won't suffice in this case, though.